content/en/blog/2024/prometheus-compatibility-survey/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Insights from the Prometheus Compatibility Survey
+linkTitle: Prometheus Compatibility Survey
+date: 2024-06-10
+author: '[David Ashpole](https://github.com/dashpole) (Google)'
+cSpell:ignore: Ashpole
+---
+
+Prometheus and OpenTelemetry are two of the most active and popular projects in
+the CNCF observability landscape. The two communities have been working together
+since the early days of OpenTelemetry to improve the compatibility between the
+two projects. The OpenTelemetry Prometheus working group has been leading this
+effort, with active participation from maintainers from both OpenTelemetry and
+Prometheus.
+
+At this point, there is a detailed, experimental specification describing how to
+convert between the OpenTelemetry metrics format and the Prometheus metrics
+format. It has been used to implement Prometheus (pull) exporters for
+OpenTelemetry SDKs, OTLP export from Prometheus libraries, OTLP ingestion for
+the Prometheus server, and the OpenTelemetry Collector's Prometheus Receiver and
+Prometheus exporters.
+
+As we consider stabilizing the compatibility specification, we decided to run a
+survey with the help of the OpenTelemetry End User SIG to collect feedback on
+the current state of compatibility between the projects, and try to understand
+user sentiment on a few of the remaining open questions. In particular, we
+wanted feedback on the translation for metric names, as we want to be sure
+before we make any changes.
+
+The survey received 86 responses, and contained many helpful pieces of feedback.
+Than you to everyone that participated!
+
+## Overall Takeaways
```

Comment:
```suggestion
## Overall takeaways
```

We use sentence case for headings. Please edit all headings accordingly — thanks!